api/src/estimate/estimate.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { EstimateController } from './estimate.controller';
+
+describe('EstimateController', () => {
+  let controller: EstimateController;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [EstimateController],
+    }).compile();
+
+    controller = module.get<EstimateController>(EstimateController);
+  });
+
+  it('should be defined', () => {
+    expect(controller).toBeDefined();
+  });
+});
```

Comment:
for now, we are not focusing on the tests, I want you to get familiar with the framework and achieve some results with it, so just keep it in your mind as a note, this is not what the tests should look like, don't change it, for now, we will talk about the testing subject later.

Reply:
test files are auto generated with controllers .. and i keep them cuz i know that in some point we will need them